op-node/flags/flags.go

```diff
@@ -179,6 +179,13 @@ var (
 		Value:    20,
 		Category: L1RPCCategory,
 	}
+	L1CacheSize = &cli.IntFlag{
```

Comment:
There is `cli.Uint64Flag`, to make an unsigned int flag. That avoids the `< 0` check in the code later on.

Reply:
changed to `Uint`